automation/smoke-test-firefox-ios.py

```diff
@@ -47,8 +47,6 @@
 if not Path(repo_path, "Carthage").exists():
     step_msg("Carthage folder not present. Running the firefox-ios bootstrap script")
     run_cmd_checked(["./bootstrap.sh"], cwd=repo_path)
-step_msg("Running carthage substitution script")
-run_cmd_checked(["./appservices_local_dev.sh", "enable", find_app_services_root()], cwd=repo_path)
```

Comment:
I was thinking about this, this script right now won't run against a local version of this, so it's a little deceiving - until we have a similar `appservices_local_dev` script for the swift packages, should we print an error message here, or disable the script or something?

Reply:
Hmmm that's a good point that it's essentially not doing anything to help smoke test against a local copy of a-s. Will have to think about this script more to see how to make it clearer what to do (or point them to the local testing of SPM for a-s for now)

Reply:
Makes sense! (look at me replying to a message 3 months ago 😆) should we also remove smoke testing from our [steps in circle ci](https://github.com/mozilla/application-services/blob/main/.circleci/config.yml#L427-L433)?